src/fetch.js

```diff
@@ -5,15 +5,17 @@ import pluralize from 'pluralize'
 module.exports = async ({
   apiURL,
   contentType,
+  singleType,
   jwtToken,
   queryLimit,
-  reporter,
+  reporter
 }) => {
   // Define API endpoint.
-  const apiBase = `${apiURL}/${pluralize(contentType)}`
+  let apiBase = singleType ? `${apiURL}/${singleType}` : `${apiURL}/${pluralize(contentType)}`
+  
   const apiEndpoint = `${apiBase}?_limit=${queryLimit}`
```

Comment:
Can you remove the limit if you are not calling the fetch function with queryLimit (e.g singleType send a ?_limit=undefined)

Reply:
The plugin uses a default of 100. I was not passing the query limit option from the gatsby-node into the singleTypes fetch data call. Both content types and single types are defaulting to 100 now with latest commit.